font-codegen/src/fields.rs

```diff
@@ -232,6 +245,16 @@ impl Fields {
     }
 }
 
+/// All the state required to generate a constructor for a table/record
+/// that includes this field.
+pub(crate) struct FieldConstructorInfo<'a> {
+    pub(crate) name: &'a syn::Ident,
```

Comment:
We have other structures that contain syn::Ident and those do not have lifetimes. Can we remove the lifetime? - it's ok if it means we copy Ident's imho.

Reply:
In general I try to borrow where doing so is A) possible and B) not overly difficult to do; I'm not sure if we have other structures that meet these criteria? Happy to change, since this is unimportant to practice, but wanted to explain my reasoning.